components/eamxx/cime_config/eamxx_buildnml.py

```diff
@@ -0,0 +1,806 @@
+#!/usr/bin/env python3
+
+"""
+Used by buildnml. See buildnml for documetation.
```

Comment:
This file has impl that used to be in buildnml. Almost nothing was meaningfully changed.